packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,121 @@
+| **selectionType** | `'single' \| 'multiple'`                  | `'single'`  | Allows multiple selections within the ChipGroup                  | ❌       |
+
+## Examples
+
+Here are a few illustrative examples showcasing the utilization of the `Chip` and `ChipGroup` components:
+
+### Basic/Uncontrolled Usage
+
+Single Chip selection:
+
+```jsx
+<ChipGroup defaultValue="1">
+  <Chip value="1">Single chip</Chip>
+  <Chip value="2">Can be selected</Chip>
+  <Chip value="3">At a time</Chip>
+</ChipGroup>
+```
+
+Multiple Chip selection:
+
+```jsx
+<ChipGroup selectionType="multiple" defaultValue={['1', '2']}>
+  <Chip value="1">Multiple chips</Chip>
+  <Chip value="2">Can be selected</Chip>
+  <Chip value="3">At a time</Chip>
+</ChipGroup>
+```
+
+### Controlled Usage
+
+```jsx
+function Single() {
+  // string value when selectionType is 'single' (default)
+  const [selected, setSelected] = useState('react');
+
+  return (
+    <ChipGroup selectionType="single" value={value} onChange={({ value }) => setSelected(value)}>
+      <Chip value="react">React</Chip>
+      <Chip value="ng">Angular</Chip>
+      <Chip value="svelte">Svelte</Chip>
+      <Chip value="vue">Vue</Chip>
+    </ChipGroup>
+  );
+}
+
+function Multiple() {
+  // array of strings value when selectionType is 'multiple'
+  const [selected, setSelected] = useState(['react']);
+
+  return (
+    <ChipGroup selectionType="multiple" value={value} onChange={({ value }) => setSelected(value)}>
+      <Chip value="react">React</Chip>
+      <Chip value="ng">Angular</Chip>
+      <Chip value="svelte">Svelte</Chip>
+      <Chip value="vue">Vue</Chip>
+    </ChipGroup>
+  );
+}
+```
+
+## Accessibility Considerations
+
+The `Chip` and `ChipGroup` components are inherently designed to uphold accessibility standards.
+They employ native `radio` and `checkbox` inputs, ensuring consistent keyboard event functionality, similar to native controls.
+This adherence to established accessibility practices is further substantiated by the following guidelines:
+
+- **Radio Accessibility** - The `Chip` component follows the [W3C ARIA Radio Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/radio/) to ensure that users utilizing assistive technologies can seamlessly interact with and understand the radio selection mechanism.
+- **Checkbox Accessibility** - Similarly, the `Chip` component aligns with the [W3C ARIA Checkbox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/) to guarantee that users with disabilities can effectively engage with the checkbox selection paradigm.
+
+These deliberate design choices manifest in a user experience that respects accessibility guidelines, enabling users of diverse abilities to interact with the components with equal ease.
+
+## Open Questions
+
+- Should we provide a `name` prop to the `ChipGroup` component, similar to the `RadioGroup` component?
```

Comment:
Probably without the `name` prop a radios won't work. 

Although we can set a default random id to the name prop internally, we should expose a `name` prop so that consumers can provide a human readable ID to the group for form submission purposes. 

This is already handled in useRadioGroup.